git/git.go

```diff
@@ -273,6 +290,17 @@ func SetupRepoAuth(logf func(string, ...any), options *options.Options) transpor
 		}
 	}
 
+	// If no path was provided, fall back to the environment variable
+	if options.GitSSHPrivateKeyBase64 != "" {
```

Comment:
It might be worth thinking about what the behavior should be if _both_ key path and key base64 is given. Do we exit immediately (invalid input)? Do we prefer one over the other? How do we inform the user?

Reply:
To be honest, I think we should error if both are provided. Removes any ambiguity.